plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerNode.java

```diff
@@ -65,12 +63,10 @@
      *
      * @param workspaceId
      *         id of workspace to unbind
-     * @param hostProjectsFolder
-     *         folder on the docker host where workspace was bound
      * @throws MachineException
      *         if error occurs on binding
      */
-    void unbindWorkspace(String workspaceId, String hostProjectsFolder) throws MachineException;
+    void unbindWorkspace(String workspaceId) throws MachineException;
```

Comment:
The same [comment](https://github.com/codenvy/che-plugins/pull/328/files#r42720194) is here